src/utils.ts

```diff
@@ -48,12 +48,16 @@ export const hasOauthClientSettings = (local = false): boolean =>
  * Should be used instead of `DOTFILE.RC?().read()`
  * @returns {Promise<ClaspToken>} A promise to get the rc file as object.
  */
-export function getOAuthSettings(local: boolean): Promise<ClaspToken> {
-  const RC = (local) ? DOTFILE.RC_LOCAL() : DOTFILE.RC;
-  return RC.read()
+export function getOAuthSettings(): Promise<ClaspToken> {
+  return DOTFILE.RC_LOCAL()
+    .read()
     .then((rc: ClaspToken) => rc)
     .catch((err: any) => {
-      logError(err, ERROR.NO_CREDENTIALS(local));
+      return DOTFILE.RC.read()
```

Comment:
I'd rather be explicit about if we're loading local or global credentials.
The previous construct was very nested and hard with nested promises.

If we can make the tests pass with the local/global RC, that would be ideal.